src/Prima/Pyxis/Table.elm

```diff
@@ -194,6 +208,18 @@ header slug name maybeTagger =
     Header <| HeaderConfiguration slug name maybeTagger
 
 
+{-| Creates a Footer.
+
+    myFooter : String -> String -> Table.Footer
+    myFooter slug content =
+        Table.footer slug content
+
+-}
+footer : Slug -> Name -> Footer msg
```

Comment:
Più che `Name` mi aspetto un `Value`. Il `Name` dell' `HeaderConfiguration` si riferisce al nome della colonna. 
In un `tfoot` le colonne sono nameless e non necessariamente collegate alle colonne del `thead`.